docs/tutorial.md

```diff
@@ -1847,8 +1847,107 @@ bone and the final locations are stored in `U`.
 
 ### Mesh Deformation with Kelvinlet
 
-!!! todo "409_Kelvinlets"
-    _Entry Missing_
+Kelvinlets[^pixar_2017] is a technique for real-time physically based volume sculpting of
+virtual elastic materials. The technique treats meshes as fluids made of
+compressible materials and deforms them by advecting points along a displacement
+field. It relies on analytical solutions to the equations of elasticity.
+
+#### A quick primer on linear elastostatics [^slaughter_2002]
+
+The equilibrium state of linear elasticity is determined by a displacement field
+$\mathbf{u} : R^3 \rightarrow  R^3$ that minimizes the elastic potential energy 
+
+$E(\mathbf{u}) = \frac{\mu}{2}\left\|\nabla\mathbf{u}\right\|^2  +
+ \frac{\mu}{2(1-2\nu)}\left\|\nabla .\mathbf{u}\right\|^2 - \langle\mathbf{b},
+ \mathbf{u}\rangle$
+
+where $\mu$ is the elastic shear modulus,  $\nu$ is the Poisson ratio, and
+$\mathbf{b}$ represents the external body forces.
+
+The first term controls the smoothness of the displacement field, the second
+term penalizes infinitesimal volume change, and the last term indicates the
+external body forces to be counteracted.
+
+One can associate the optimal displacement field with the solution to the
+critical point of the above equation, also known as the Navier-Cauchy equation:
+
+$\mu\Delta\mathbf{u} + \frac{\mu}{(1 - 2\nu)}\nabla(\nabla . \mathbf{u}) +
+\mathbf{b} = 0$
+
+The Kelvinlet is the solution to the Navier-Cauchy equation in the case of a
+concentrated body load due to a force vector $\mathbf{f}$ at a point
+$\mathbf{x}_{0}$, i.e., where $\mathbf{b}(\mathbf{x}) = \mathbf{f}
+\delta(\mathbf{x} − \mathbf{x}_{0})$ and can be written as:
+
+$\mathbf{u}(\mathbf{r}) = \left[ \frac{(a - b)}{r}I +
+\frac{\mathbf{b}}{\mathit{r}^3}\mathbf{r}\mathbf{r}^{t}\right] \equiv
+\mathbf{K}(\mathbf{r})\mathbf{f}$
```

Comment:
I think you do not explain what is `K`, and why it depends on `r` here.

Reply:
K is the kelvinlet function. Will make that explicit